### DISCUSSION TITLE
Ability to set custom input as a error target

### DISCUSSION THREAD

**Discussion by patjakubik** [Shopify/function-examples#366] [Feedback: Cart and Checkout Validation API]
We want our custom input to enter an error state if it fails validation with the Checkout Validation Function.

For example:

```
<Checkbox id="mandatory_checkbox">Mandatory checkbox</Checkbox>
```

FunctionRunResult:

```
    [
      {
        localizedMessage: "This checkbox is mandatory",
        target: "$.mandatory_checkbox",
      },
    ];
```

**Comment by gil--** [comment]
Would love to see this as well to ensure checkout is blocked for required non-native fields.
